### DISCUSSION TITLE
Docker Image not listening to ipv6

### DISCUSSION THREAD

**Discussion by cycl0ne** [kanboard/kanboard#5204] [Installation & Configuration]
### Actual behaviour

When starting the docker on my Synology, i can access the page through ipv4:port, but not in ipv6:port

### Expected behaviour

Hopefully ipv6 accessable

### Steps to reproduce

run docker, access the website from ipv6:port

### Configuration

Application version: v1.2.16
PHP version: 7.3.23
PHP SAPI: fpm-fcgi
HTTP Client: cURL
OS version: Linux 4.4.59+
Database driver: sqlite
Database version: 3.32.1
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36

**Comment by cycl0ne** [comment]
I checked your supplied nginx version with:
nginx -V 2>&1 >/dev/null | grep --color -o with-ipv6

it seems you didnt compile the ipv6 with it. so no ipv6 working.

**Comment by fguillot** [comment]
> it seems you didnt compile the ipv6 with it. so no ipv6 working.

This is the default Nginx provided with Alpine Linux.

https://github.com/kanboard/kanboard/blob/master/Dockerfile#L11-L15

Feel free to create your own Docker image.

**Comment by cycl0ne** [comment]
i will contact alpine. from their bug list, the ipv6 flag was enabled 10 years ago.

**Comment by cycl0ne** [comment]
Just an Update: here the case number from them:
https://gitlab.alpinelinux.org/alpine/aports/-/issues/12838

**Comment by cycl0ne** [comment]
ok its partly a docker problem and partly a nginx config problem. I will investigate further.

in nginx.conf the line for ipv6 is missing:
listen          [::]:80

maybe one could add it, maybe as env variable.

cheers
